scripts/build.ts

```diff
@@ -2,13 +2,9 @@ import { $ } from "bun";
 $.nothrow();
 
 const { arch, platform } = process;
-const unsupported = () => {
```

Comment:
Risc-V is on the radar and Win32 might be still out there ... what are the gains here?

Reply:
Might me being just pedantic. Will add them back once bun launches on other platforms.